src/wp-includes/block-supports/layout.php

```diff
@@ -540,6 +540,26 @@ function wp_get_layout_style( $selector, $layout, $has_block_gap_support = false
 	return '';
 }
 
+/**
+ * Generates an incremental ID that is independent per each different prefix.
+ *
+ * It is similar to `wp_unique_id`, but each prefix has it's own internal ID
+ * counter to make each prefix independent from each other. The ID starts at 1
+ * and increments on each call. The returned value is not universally unique,
+ * but it is unique across the life of the PHP process and it's stable per
+ * prefix.
+ *
+ * @param  string $prefix Prefix for the returned ID.
```

Comment:
If this is to remain as an optional parameter (see [here](https://github.com/WordPress/wordpress-develop/pull/5528#discussion_r1366331137) first), it should be documented as such:


```suggestion
 * @param  string $prefix Optional. Prefix for the returned ID. Default empty string.
```

Reply:
Fixed in [2d3db44](https://github.com/WordPress/wordpress-develop/pull/5528/commits/2d3db4418b82b8860fd10109cdb9fefaa4b2be09). 👍